src/osp/Active/ActiveScene.h

```diff
@@ -206,31 +202,13 @@ class ActiveScene
 
     bool system_machine_it_valid(MapSysMachine_t::iterator it);
 
-    /**
-     *
-     * @tparam T
-     */
-    template<class DYNSYS_T, typename... Args>
-    DYNSYS_T& dynamic_system_create(Args &&... args);
-
+    template<typename... ARGS_T>
+    void debug_update_add(ARGS_T &&... args)
+    { m_updateHandles.emplace_back(std::forward<ARGS_T>(args)...); }
 
-    /**
-     * Find a registered IDynamicSystem by name. This accesses a map.
-     * @param name [in] Name used as a key
-     * @return Iterator to specified IDynamicSys
-     */
-    MapDynamicSys_t::iterator dynamic_system_find(std::string_view name);
-
-    /**
-     * Find a registered IDynamicSystem by type, and cast it to SYSTEM_T. This
-     * accesses a map. If a system isn't found, this will assert.
-     * @tparam SYSTEM_T Type of registered IDynamicSys
-     * @return Reference to specified IDynamicSys
-     */
-    template<class SYSTEM_T>
-    SYSTEM_T& dynamic_system_find();
-
-    bool dynamic_system_it_valid(MapDynamicSys_t::iterator it);
+    template<typename... ARGS_T>
+    void debug_render_add(ARGS_T &&... args)
+    { m_renderHandles.emplace_back(std::forward<ARGS_T>(args)...); }
```

Comment:
I believe that emplace back returns a reference.

You might as well return what it returns. Something might want to use the reference in the future.